network/load-balancer/how-to/manage-frontends-and-backends.mdx

```diff
@@ -54,7 +54,7 @@ When you [create your Load Balancer](/network/load-balancer/how-to/create-load-b
 3. Click the **Backends** tab.
 4. Click the green **+** button.
     <Lightbox src="scaleway-add-backends.webp" />
-5. Enter a name, protocol, port, proxy, health check type,  server IP(s) and advanced settings as required for your new backend.
+5. Enter a name, protocol, port, proxy, health check type,  server IP(s) and advanced settings as required for your new backend. For help configuring your frontend, see our [dedicated reference documentation](/network/load-balancer/reference-content/configuring-load-balancer)
```

Comment:
> For help configuring your frontend, ...

We are in the backend section, please replace frontend with backend :)

Reply:
thanks, corrected